manifests/s/SomePythonThings/ElevenClock/3.4.0/SomePythonThings.ElevenClock.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SomePythonThings.ElevenClock
+PackageVersion: 3.4.0
```

```suggestion
PackageVersion: 3.4.0.1
```
Weird version number issue..